cli/compose/convert/service.go

```diff
@@ -36,9 +37,19 @@ func Services(
 		if err != nil {
 			return nil, errors.Wrapf(err, "service %s", service.Name)
 		}
-		configs, err := convertServiceConfigObjs(client, namespace, service.Configs, config.Configs)
-		if err != nil {
-			return nil, errors.Wrapf(err, "service %s", service.Name)
+		var configs []*swarm.ConfigReference
+		if len(service.Configs) > 0 {
```

Comment:
Oh, sorry, didn't see your comment.

I also was just looking how we handle other API endpoints that are not supported yet, and it looks like we have code to handle that, e.g.; https://github.com/moby/moby/blob/745795ef2e0089c5001e5a2fc7ba8c1ab0234857/client/container_prune.go#L16-L18

But looks like we didn't add such handling to the secrets and config API